expyfun/_utils.py

```diff
@@ -426,12 +418,21 @@ def verbose_dec(function, *args, **kwargs):
         return ret
 
 
+def _has_avbin():
+    try:
+        from pyglet.media.avbin import AVbinSource
+        del AVbinSource
+        return True
+    except ImportError:
+        return False
+
+
 _is_appveyor = (os.getenv('APPVEYOR', 'False').lower() == 'true')
 requires_pandas = skipif(has_pandas is False, 'Requires pandas')
 requires_h5py = skipif(has_h5py is False, 'Requires h5py')
 requires_joblib = skipif(has_joblib is False, 'Requires joblib')
 requires_opengl21 = skipif(_is_appveyor, 'Appveyor OpenGL too old')
-requires_avbin = skipif(has_avbin is False, 'Requires AVbin')
+requires_avbin = skipif(_has_avbin() is False, 'Requires AVbin')
```

Comment:
this brings up another question: in the mne function you linked to, it says `skipif(not condition, ...)` whereas the functions in expyfun._utils say `skipif(condition is False, ...)`.  I was initially going to change those to `not condition` but decided you must have had a reason to do the more stringent `condition is False` test.  But seeing that mne-python does it the way I would have done it, I thought I'd raise the question.